doc/cephfs/fs-volumes.rst

```diff
@@ -790,32 +790,38 @@ pinned to one of the available ranks on the file system.
 Subvolume quiesce
 -----------------
 
-It may be needed to pause IO to a set of subvolumes of a given volume (file system).
-A good example of such case is a consistent snapshot spanning multiple subvolumes.
-The task arises often in an orchestrated environment such as Kubernetes, where a single deployed application
-can work with many mounted subvolumes across several hosts. When a snapshot of such a system is needed,
-the application may not find the result consistent unless the snapshots were taken
-during an active write pause.
+Distributed applications that run against different mount points of the same Ceph file system
+are facing a challenge of consistent snapshots for backups and disaster recovery. The challenge is due to
+the fact that CephFS snapshots dont provide strong consistency guarantees when there are multiple
+clients performing writes. Even when the application is using file system flushes to synchronize
+checkpoints across its distributed components, there aren't guarantees that all acked writes will
+be part of a given snapshot.
 
-The `volumes` plugin provides a tool to initiate and await such a pause across a set of subvolumes:
+The subvolume quiesce feature has been developed to provide enterprise-level consistency guarantees
+for multi-client applications working with one or more subvolumes. The feature allows to pause IO
+to a set of subvolumes of a given volume (file system). By reaching such a pause across all clients
+we can guarantee that any persistent checkpoints reached by the application will be recoverable
+from the snapshot group made during the pause. We refer to the pause as `quiesce`
+
+The `volumes` plugin now provides a tool to initiate and await the pause for a set of subvolumes:
```

Comment:
s/now//

(delete "now"--This document is going to live long enough that soon enough, "now" won't refer to what it referred to today.)

Reply:
done